webapp/src/stores/model.ts

```diff
@@ -68,6 +68,7 @@ export interface Task extends Item {
     fileRequirements: FileRequirements;
     xmlTemplate: string;
     rpmLimitation: RPMLimitation;
+    timeoutInHours: string;
```

Comment:
why string? why not number?

Reply:
In all numbers I had found in current front end, all are using string. There is not even number validator. Thats why String.  Do ou mind to provide more guidance?
Also I would sugest to change at least this and versionNumber to string as separate commit, as it is transparent change.

Reply:
ou, I thought versionNumber is...number :D ok, separate commit then